src/io/HMpsFF.cpp

```diff
@@ -1723,4 +1930,10 @@ typename HMpsFF::Parsekey HMpsFF::parseSos(const HighsLogOptions& log_options,
   return HMpsFF::Parsekey::kFail;
 }
 
+bool HMpsFF::allZeroed(const std::vector<double> value) {
```

Comment:
```suggestion
bool HMpsFF::allZeroed(const std::vector<double>& value) {
```
(do not copy vector)